code/mission/missionbriefcommon.cpp

```diff
@@ -899,10 +907,24 @@ void brief_render_fade_outs(float frametime)
 				continue;
 			}
 
-			bm_get_info( fi->fade_anim.first_frame, &w, &h, NULL);
+			float scaled_w, scaled_h;
+
+			bm_get_info( fi->fade_anim.first_frame, &w, &h, nullptr);
 			float screenX = tv.screen.xyw.x;
 			float screenY = tv.screen.xyw.y;
-			gr_unsize_screen_posf( &screenX, &screenY, NULL, NULL, GR_RESIZE_MENU_NO_OFFSET );
+
+			int this_resize = bscreen.resize;
+			if (bscreen.resize == GR_RESIZE_MENU) {
+				this_resize = GR_RESIZE_MENU_NO_OFFSET;
+			}
```

Comment:
What _exactly_ does that entail? Will this possibly break certain cases here? Will some stuff be now potentially rendered without offset it may need?

Reply:
I honestly don't remember exactly what the differences are. I remember testing it waaaaay back when I made the methods that allowed SCPUI to render the briefing map. Either way the regular icons do exactly this already so this is just to match that.